### PR TITLE
Fixes #9400 - configure networking in finish template

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: app/views/foreman/unattended/finish-katello.erb [hunk 1]
```diff
@@ -11,6 +11,11 @@ oses:
 - Fedora 19
 - Fedora 20
 %>
+<% if @host.subnet.respond_to?(:dhcp_boot_mode?) -%>
+<%= snippet 'kickstart_networking_setup' %>
+service network restart
```
Comment (jlsherrill) [inline app/views/foreman/unattended/finish-katello.erb]: is a service network restart safe to do during a kickstart's %post section?

Comment (jlsherrill) [inline app/views/foreman/unattended/finish-katello.erb]: paging @ares

Comment (stbenjam) [inline app/views/foreman/unattended/finish-katello.erb]: @jlsherrill Finish script's aren't used during kickstart (despite the snippet's name that's being included), it happens via SSH.  It should be safe to do if networking info is correct

Comment (ares) [inline app/views/foreman/unattended/finish-katello.erb]: fully agree with what @stbenjam says

Comment (jlsherrill) [inline app/views/foreman/unattended/finish-katello.erb]: Ahhh +1 for me then.

**Comment by jlsherrill** [comment]
ACK from me unless you have objects @stbenjam

**Comment by stbenjam** [comment]
Oh sorry, yes, good for me - it needs backporting to 2.2.

**Pull request merged**